### PR TITLE
Create Red Wave.xaml

### PR DESCRIPTION
**Pull request by Rasmagoric**
Reddish / Orange skin
image

### CODE REVIEW

File: CustomSkinsPackage/Red Wave.xaml [hunk 1]
```diff
@@ -6,12 +6,17 @@
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
 
+            <ResourceDictionary.MergedDictionaries>
+                <Windows10version1809:ColorPaletteResources ChromeHigh="#d45332"/>
+            </ResourceDictionary.MergedDictionaries>
```
Comment (yaichenbaum) [inline CustomSkinsPackage/Red Wave.xaml:11]: What's the goal of this?

Comment (Rasmagoric) [reply CustomSkinsPackage/Red Wave.xaml:11]: > 
> 
> What's the goal of this?

The lines that separate the columns in column view. It's the only I found to change them.

**Review by yaichenbaum — APPROVED** [review]
LGTM

File: CustomSkinsPackage/Red Wave.xaml (hunk 1 above)
Comment (yaichenbaum) [reply CustomSkinsPackage/Red Wave.xaml:11]: Being that Files doesn't target lower than 1809, do you need this to be conditional? Fyi @winston-de

Comment (winston-de) [reply CustomSkinsPackage/Red Wave.xaml:11]: No, the conditional isn't necessary.

File: CustomSkinsPackage/Red Wave.xaml [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:BelowWindows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract, 7)"
+    xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)">
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+
+            <ResourceDictionary.MergedDictionaries>
+                <Windows10version1809:ColorPaletteResources ChromeHigh="#d45332"/>
+            </ResourceDictionary.MergedDictionaries>
```
Comment (yaichenbaum) [inline CustomSkinsPackage/Red Wave.xaml:11]:
```suggestion
            <Color x:Key="ChromeHigh">#d45332</Color>
```

File: CustomSkinsPackage/Red Wave.xaml [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+                FallbackColor="{ThemeResource SolidBackgroundFillColorBase}"
+                TintColor="{ThemeResource SolidBackgroundFillColorBase}"
+                TintOpacity="0.98" />
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="Light">
+
+            <ResourceDictionary.MergedDictionaries>
+                <Windows10version1809:ColorPaletteResources ChromeHigh="#d45332"/>
+            </ResourceDictionary.MergedDictionaries>
```
Comment (yaichenbaum) [inline CustomSkinsPackage/Red Wave.xaml:135]:
```suggestion
            <Color x:Key="ChromeHigh">#d45332</Color>
```

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Pull request merged**